pymongo/collection.py

```diff
@@ -1475,13 +1536,16 @@ def estimated_document_count(self, **kwargs: Any) -> int:
 
         :Parameters:
           - `**kwargs` (optional): See list of options above.
+          - `comment` (optional): A user-provided comment to attach to this
```

Comment:
comment's doc should appear before `**kwargs`.

Reply:
Done.